AsBuiltReport.VMware.vROPs.json

```diff
@@ -0,0 +1,26 @@
+{
+    "Report": {
+        "Name": "vRealize Operations Manager As Built Documentation",
+        "Version": "1.0",
+        "Status": "Released"
+    },
+    "Options": {
+        "AuthSource": "local",
+        "AlertFilter": "*"
+    },
+    "HealthCheck": {},
+    "InfoLevel": {
+        "_comment_": "0 = Disabled, 1 = Summary, 2 = Informative, 3 = Detailed, 4 = Adv Detailed, 5 = Comprehensive",
```

Comment:
Do you intend on having 5 InfoLevels? Only use the number of InfoLevels you need or intend on building.

Reply:
Have set it to max of 3.